src/lib/core/typography/_typography-utils.scss

```diff
@@ -31,7 +36,8 @@
 
 // Outputs the shorthand `font` CSS property, based on a set of typography values. Falls back to
 // the individual properties if a value that isn't allowed in the shorthand is passed in.
-@mixin mat-typography-font-shorthand($font-size, $font-weight, $line-height, $font-family) {
+@mixin mat-typography-font-shorthand(
```

Comment:
The `mat-typography-font-shorthand` is only for generating shorthand `font` properties. The letter spacing should go in the `mat-typography-level-to-styles` mixin.

Reply:
Done.